src/overlays/actors/ovl_En_Water_Effect/z_en_water_effect.c

```diff
@@ -12,10 +20,10 @@
 
 #define THIS ((EnWaterEffect*)thisx)
 
-void EnWaterEffect_Init(Actor* thisx, PlayState* play);
-void EnWaterEffect_Destroy(Actor* thisx, PlayState* play);
-void EnWaterEffect_Update(Actor* thisx, PlayState* play);
-void EnWaterEffect_Draw(Actor* thisx, PlayState* play);
+void ENWATEREFFECT_TYPE_Init(Actor* thisx, PlayState* play);
+void ENWATEREFFECT_TYPE_Destroy(Actor* thisx, PlayState* play);
+void ENWATEREFFECT_TYPE_Update(Actor* thisx, PlayState* play);
+void ENWATEREFFECT_TYPE_Draw(Actor* thisx, PlayState* play);
```

Comment:
Looks like some find and replace went wrong

Reply:
That is *exactly* what happened. I'll fix it.